lib/subscribe-to-sync-events.js

```diff
@@ -0,0 +1,15 @@
+module.exports = subscribeToSyncEvents
+
+var toObject = require('./utils/to-object')
+
+function subscribeToSyncEvents (syncApi, emmiter) {
+  syncApi.on('push', emitHoodieObject.bind(emmiter, 'push'))
+  syncApi.on('pull', emitHoodieObject.bind(emmiter, 'pull'))
+  syncApi.on('sync', emitHoodieObject.bind(emmiter, 'sync'))
+  syncApi.on('connect', emitHoodieObject.bind(emmiter, 'connect'))
+  syncApi.on('disconnect', emitHoodieObject.bind(emmiter, 'disconnect'))
```

Comment:
I think this will break, as `connect` / `disconnect` don't pass an object. So maybe in emitHoodieObject emit the event only if object is undefined?